### PR TITLE
doc: refresh the Clear Linux OS on GCP guide

### PR DESCRIPTION
**Pull request by ttzeng**
Update the Clear Linux OS on Google Cloud Platform guide with:

- Screenshots for the new [Storage bucket creation](https://cloud.google.com/storage/docs/creating-buckets).
- Remove the [black dot in the username limitation](https://github.com/clearlinux/distribution/issues/829) warning.

Signed-off-by: Tonny Tzeng <tonny.tzeng@intel.com>

### CODE REVIEW

File: source/get-started/cloud-install/gce.rst [hunk 1]
```diff
@@ -229,12 +229,10 @@ Setup |CL| VM on GCP
...
         Figure 18: Set SSH key for remote login
 
-     .. warning::
+     .. note::
 
         The username is assigned from characters preceding ``@`` in the email
-        address, included in the SSH key. The dot symbol "." is not allowed,
-        because it is an invalid character while creating user accounts in
-        |CL|.
+        address, included in the SSH key.
```
Comment (mvincerx) [inline source/get-started/cloud-install/gce.rst]: Hi @ttzeng. On GCP, has this changed to allow the '.' (dot) in the username?

Comment (ttzeng) [reply source/get-started/cloud-install/gce.rst]: @mvincerx , the limitation of '.' (dot) in the username was imposed in Clear Linux, and it's fixed in the later releases. (ref: [Issue#829](https://github.com/clearlinux/distribution/issues/829) of CL distribution repo).

Comment (mvincerx) [reply source/get-started/cloud-install/gce.rst]: Thanks for the clarification.

**Comment by mvincerx** [comment]
@ttzeng. Will you please resubmit this as a branch and not as your "master"? 
I know you're aware of this. In case you need a refresher on contributing, see [this link](https://docs.01.org/clearlinux/latest/collaboration/collaboration.html#contribute-via-github).

**Pull request closed without merging**